### PR TITLE
Actuator endpoint skipped by default

### PR DESCRIPTION
**Pull request by larchanjo**
I've fixed it https://github.com/opentracing-contrib/java-spring-web/issues/98, is it ok?

### CODE REVIEW

File: opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java [hunk 1]
```diff
@@ -115,6 +122,17 @@ public void testExcluded() throws InterruptedException {
         assertThat(Mockito.mockingDetails(mockDecorator2).getInvocations()).hasSize(0);
     }
 
+    // Test that /actuator/info is excluded due to the default skipPattern
+    @Test
+    public void testActuatorExcluded() throws InterruptedException {
```
Comment (geoand) [inline opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java]: Can you please also change the name of `testExcluded` to `testInfoExcluded`?

**Comment by geoand** [comment]
Hi @larchanjo, thanks for the contribution! It looks good, I just added one small comment.

File: opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java (hunk 1 above)
Comment (larchanjo) [reply opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java]: Yes of course, It's done, thank you for the feedback.

Comment (geoand) [reply opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/ServerTracingAutoConfigurationDefaultsTest.java]: Thanks! I'll merge this once CI goes green.

**Comment by geoand** [comment]
Thanks again @larchanjo !

**Pull request merged**